server/huntflow_reloaded/models.py

```diff
@@ -4,6 +4,18 @@
 
 DB = Gino()
 
+class User(DB.Model):
+    """ User for authorization """
+
+    __tablename__ = 'users'
+
+    id = DB.Column(DB.Integer(), primary_key=True, autoincrement=True) # pylint: disable=maybe-no-member
+
+    email = DB.Column(DB.String()) # pylint: disable=maybe-no-member
+    password = DB.Column(DB.String()) # pylint: disable=maybe-no-member
```

Comment:
PEP8: inline comments should be separated by at least two spaces from the statement.